src/web/lib/acquisitions.ts

```diff
@@ -0,0 +1,49 @@
+import {OphanComponentType} from "@root/src/web/browser/ophan/ophan";
+import {constructQuery} from "@root/src/lib/querystring";
```

Comment:
noticed you have some styling issues that are not handled by prettier. I had this issue as well, all i needed was to update VSC and just enable prettier on save

Reply:
I ran `make fix`, should this do all the formatting? I don't use VSC